src/main/kotlin/fi/metatavu/timebank/api/impl/VacationsApi.kt

```diff
@@ -0,0 +1,54 @@
+package fi.metatavu.timebank.api.impl
+
+import fi.metatavu.timebank.api.controllers.VacationController
+import fi.metatavu.timebank.api.impl.translate.VacationRequestTranslator
+import fi.metatavu.timebank.model.VacationRequest
+import javax.enterprise.context.RequestScoped
+import fi.metatavu.timebank.spec.VacationsApi
+import java.time.LocalDate
+import java.util.*
+import javax.inject.Inject
+import javax.ws.rs.core.Response
+
+/**
+ * API implementation for Vacations API
```

Comment:
There are already vacations used as a term within dailyentries, maybe this should have a consistent naming of VacationRequests.